index.js

```diff
@@ -169,4 +169,30 @@ const isEmptyMapOrSet = x => (is.map(x) || is.set(x)) && x.size === 0;
 
 is.empty = x => !x || isEmptyStringOrArray(x) || isEmptyObject(x) || isEmptyMapOrSet(x);
 
+is.any = function (predicate, values) {
+	let ret = false;
+	values = Array.prototype.slice.call(arguments, 1);
```

Comment:
Generally, just a note, don't use `arguments`, ES5 has given has enough ways to deprecate usage pf `arguments`.

If you really end up having to use `arguments`, make sure you document it well.

Reply:
What would I use instead of `arguments` if I can't use `...values`?

Reply:
Notice the comment I've made above. I wrote it before I've went over the commits.

Reply:
@gioragutt oh, gotcha!